src/lib/updating-element.ts

```diff
@@ -215,6 +217,14 @@ export abstract class UpdatingElement extends HTMLElement {
 
   static properties: PropertyDeclarations = {};
 
+  /**
+   * Array of styles to apply to the element. The styles should be defined
+   * using the `css` tag function.
+   */
+  static get styles() {
```

Comment:
Add type of `Array<CSSStyleSheet|LiteralString>`

Reply:
Added specific type for this and used it here.